packages/app-headless-cms/src/admin/components/ContentModelForm/ContentModelFormRender/fields/Time.tsx

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import Input from "./Input";
+
+const Time = (props) => {
+  // "09:00:00"
```

Comment:
:+1: 
Yeah. It would be possible provided we make the change you suggested on the UI side.
Because the API is already designed with `seconds` value in mind